apps/nx-workshop-e2e/tests/nx-workshop.spec.ts

```diff
@@ -59,9 +57,7 @@ function createNewWorkspace() {
   removeSync(tmpProjPath());
   // create new workspace
   execSync(
-    `node ${require.resolve(
-      'nx'
-    )} new ${scope} --nx-workspace-root=${localTmpDir} --no-interactive --skip-install --collection=@nx/workspace --npmScope=${scope} --preset=apps --packageManager=npm`,
+    `npx create-nx-workspace ${scope} --no-interactive --skip-install --preset=apps --packageManager=npm --ci=skip`,
```

Comment:
Is `--preset=apps` a noop now?

I kind of expected it to create an `apps` and `libs` dir but it seems to be equivalent to running it without anything.

I guess that would make sense since the `workspaceLayout` config is deprecated?

Reply:
Yes, we can drop that.